internal/services/notification/db/objects/objects.go

```diff
@@ -0,0 +1,37 @@
+package objects
+
+import (
+	"agola.io/agola/internal/sqlg"
+)
+
+const (
+	Version = uint(1)
+)
+
+const TypesImport = "agola.io/agola/services/notification/types"
+
+var AdditionalImports = []string{"time"}
+
+var ObjectsInfo = []sqlg.ObjectInfo{
+	{
+		Name: "RunWebhook", Table: "runwebhook",
+		Fields: []sqlg.ObjectField{
+			{Name: "UUID", Type: "string"},
+			{Name: "Delivered", Type: "bool"},
```

Comment:
Delivered here doesn't makes sense if you already have a RunWebhookDelivery table (that could also contains multiple deliveries for every webhook).